docs/tokens/esdt-tokens.mdx

```diff
@@ -330,6 +330,8 @@ Sending a transaction containing both an ESDT transfer _and a method call_ allow
 There is also the possibility to perform multiple tokens transfers in a single bulk. This way, one can send (to a single receiver) multiple
 fungible, semi-fungible or non-fungible tokens via a single transaction.
 
+It is also possible to send ESDT tokens to a contract as part of a method call.
```

Comment:
maybe define a new section for multi transfers with sc calls? similar to this one: 
https://docs.multiversx.com/tokens/esdt-tokens#transfers-to-a-smart-contract

Reply:
Sure, I thought about it too.
Something like `Multiple tokens transfer to a smart contract` without copying the general information (like: Payable versus non-payable) from `Transfers to a smart contract` with new code example, right?

Reply:
I think those information can be copied as well. Even if the info is duplicated, it's more intuitive if one receives a link to that docs section. Also, place it after the multi tokens transfer definition

Reply:
added a new section

Reply:
🚀